partcad/tests/unit/test_part.py

```
@@ -49,6 +49,26 @@
     wrapped = asyncio.run(bolt.get_wrapped())
     assert wrapped is not None
 
+def test_part_get_brep_1():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_part_get_brep_1`

Add return type annotation: `None`

(ANN201)